src/main/connect/connectionManager.ts

```diff
@@ -748,19 +782,113 @@ export class ConnectionManager implements ExtensionComponent, vscode.Disposable
         progress: XrayScanProgress,
         checkCanceled: () => void,
         project: string,
-        watches: string[]
+        watches: string[],
+        msi?: string,
+        packageType?: string
     ): Promise<IGraphResponse> {
-        let policyMessage: string = '';
+        let client: JfrogClient = this.createJfrogClient();
+        let scanService: XrayScanClient = client.xray().scan();
+        let technology: string[] | undefined;
+        if (msi && msi !== '' && packageType) {
+            scanService = client.xsc().scan();
+            technology = [packageType];
+        }
+        this._logManager.logMessage(this.getScanLog(graphRequest.component_id, project, watches, msi, packageType), 'DEBUG');
+        return await scanService.graph(graphRequest, progress, checkCanceled, project, watches, msi, technology);
+    }
+
+    private getScanLog(componentId: string, project: string, watches: string[], msi?: string, packageType?: string): string {
+        let service: string = 'Xray';
+        let message: string = '';
+        if (msi && msi !== '') {
+            service = 'XSC';
```

Comment:
If we're printing this, I wonder if we should add (Xray-Source-Control) to the service name. WDYT?

Reply:
changed to Xray-Source-Control